src/NoFrixion.MoneyMoov/Permissions/Permissions.cs

```diff
@@ -44,8 +44,13 @@ public enum AccountPermissions : ulong
     CanViewTransactions = 1L << 14,
     
     // Statements
+    [Obsolete("Use CanExportData instead.")]
     CanCreateStatements = 1L << 15,
+    
+    [Obsolete("Use CanExportData instead.")]
```

Comment:
Is there a customer already using these `CanCreateStatements` and `CanViewStatements` in Prod? 😮

Reply:
Not yet. I'll remove them in another PR

Reply:
Ah okay, brilliant! Thanks